hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/StandaloneEncryptedFileSystem.java

```diff
@@ -164,6 +164,9 @@ private static Function<Path, Path> setSchemeFunc(final String scheme) {
 
     private static Function<URI, URI> setUriSchemeFunc(final String scheme) {
         return uri -> {
+            if (uri.getScheme() == null) {
```

Comment:
Two quick things:
- Do you have an example where things have broken when this scheme is set?
- Can we write a test that illustrates this behavior in action?